app/js/controllers.js

```diff
@@ -4515,6 +4519,17 @@ angular.module('myApp.controllers', ['myApp.i18n'])
         }
         $rootScope.$broadcast('settings_changed')
       }
+
+      $scope.toggleDarkmode = function () {
```

Comment:
```
$scope.setTheme = function () {
  if (!$scope.theme) {
    Storage.remove('active_theme');
  } else {
    Storage.set({active_theme: 'default'});
  }
  $rootScope.$broadcast('settings_changed');
  LayoutSwitchService.switchLayout(false);
}
```
This may not work exactly as-is, but you get the idea.